src/login/LoginPage.tsx

```
@@ -0,0 +1,78 @@
+import { Button } from "@/components/ui/button"
+import { Input } from "@/components/ui/input"
+import { Card, CardContent } from "@/components/ui/card"
+import { useNavigate } from "react-router-dom"
+import { toast } from "react-toastify"
+
+export function LoginPage() {
+    const navigate = useNavigate()
+
+    function handleSubmit(e: React.FormEvent) {
```

ESLint

[error] Delete `··`

(prettier/prettier)